app/models/bulkrax/importer.rb

```diff
@@ -97,7 +97,7 @@ def schedulable?
     end
 
     def current_run
-      @current_run ||= self.importer_runs.create!(total_work_entries: self.limit || parser.total, total_collection_entries: parser.collections_total)
```

Comment:
it seems like this will break total reporting for anything that is not a zip file.

Reply:
good point. I made this a conditional instead and tested each of the following successfully:
- import a csv of a work with no files
- import of a csv of a work with remote files
- import a zip of a csv with a "files" folder (local file)
- export by work type
- export by collection
- export by importer
- round trip of work type export
- round trip of collection export
- round trip of an importer export